static/main.html

```diff
@@ -8,7 +8,8 @@
 {{end}}
 
 {{if .Video}}
-<script type="application/javascript" src="/static/js/flv.min.js"></script>
+<script type="application/javascript" src="/static/js/mpegts.js"></script>
+<script type="application/javascript" src="https://cdn.jsdelivr.net/npm/hls.js@1.6.12/dist/hls.min.js"></script>
```

Comment:
Idk, does that present a vulnerability to a supply chain attack?

Reply:
iirc, jsdeliver doesn't let you reupload existing versions, so it would only be a problem if it was something like `@1.6` or `@1.6.x`. I don't know how the versioning numbers work exactly to pin only to a minor version.

Reply:
We can always implement our own hash check on the file if we want since we're specifying a minor revision :)

Reply:
And we'd become responsible for maintaining that code and fixing any vulnerability in it, instead of just bumping the version number as needed :)

While supply chain attacks are certainly a legitimate concern, if there's a secure way to use a CDN for package imports I will always prefer it over bloating a project and taking on the burden of maintaining code we didn't write ourselves